cmd/integration/commands/idx_optimize.go

```diff
@@ -0,0 +1,199 @@
+package commands
+
+import (
+	"io/fs"
+	"log"
```

Comment:
please use `"github.com/erigontech/erigon-lib/log/v3"` package for logging. 
can be used by `log.Info("message", "some_key", "some_value", "another_key", "another_value")`.
and remove `log` package use

Reply:
done